src/mtev_cluster.c

```diff
@@ -627,7 +630,7 @@ int mtev_cluster_update_internal(mtev_conf_section_t cluster) {
      strlcpy(bufstr, "200", sizeof(bufstr));
   }
   period = strtoll(bufstr, &endptr, 10);
-  if(*endptr || period < 0 || period > 5000) {
+  if(*endptr || period < 0 || period > 30000) {
```

Comment:
That's a really high period for a cluster. 30s?  I think 5s is a reasonably high bound for period.  What's the motivation here?

Reply:
I upped this for debugging purposes. Logs quickly get spammed if you have a lot of debugging statements firing at every 5sec. Clearly you don't want to run like this in prod.

Also, I don't see harm in allowing such periods.

Reply:
Clusters with such high periods tend to behave in ways very surprising to the operator.   The dial on the instrument shouldn't go far beyond useful.  Our period's in prod should likely be around 500ms.

Reply:
OK. Reverting.